utils/hct/hctstart.cmd

```diff
@@ -115,21 +115,23 @@ echo  hctstart [-x86 or -x64] [path-to-sources] [path-to-build]
 echo.
 goto :eof
 
-:findcmake
+:findcmake 
+call :ifexistaddpath "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
```

Comment:
Why just the community addition? Should we also do it for Professional, and Enterprise?